torch/_dynamo/variables/torch.py

```diff
@@ -819,147 +831,144 @@ def call_function(
             if result:
                 return result
 
-        if can_dispatch_torch_function(tx, args, kwargs):
```

Comment:
Note: Removing this conditional is what causes the large change below (removing a large indent)